### ISSUE TITLE
TimelineGrid updateWidth performance gain

### ISSUE THREAD

**Issue by MartijnWelker** [fullcalendar/fullcalendar-scheduler#277]
By rewriting

```
this.headColEls.slice(0, -1).add(this.slatColEls.slice(0, -1)).width(nonLastSlotWidth);
```

to 

```
this.headColEls.slice(0, -1).add(this.slatColEls.slice(0, -1)).css('width', nonLastSlotWidth);
```

you get up to a 90% performance gain in large date ranges (dropped from 800ms+ to 50ms) because jQuery's `.width` [also calculates border-box values](http://api.jquery.com/width/#entry-longdesc), and `.css` does not.

Since this function gets called quite a few times (render, rerender, destroy) it would be nice if this was default behaviour or configurable.

Using `.width`: 
<img width="137" alt="screen shot 2017-03-13 at 16 18 18" src="https://cloud.githubusercontent.com/assets/6674662/23860940/c89ed196-0808-11e7-81d3-0d44532f98c2.png">

Using `.css`
<img width="121" alt="screen shot 2017-03-13 at 16 18 43" src="https://cloud.githubusercontent.com/assets/6674662/23860944/cb127f86-0808-11e7-8f3d-cd8a28921ba6.png">

**Comment by arshaw** [comment]
cool, thanks. i'll look for other instances of this

**Comment by MartijnWelker** [comment]
`renderFgSegsInContainers` also calls `append` a ton of times (which causes a repaint for every call), maybe this can be rewritten to use a map and call append only once for every containerEl?

**Comment by MartijnWelker** [comment]
`updateSegFollowers` also benefits from a major performance increase if you set the position using a css class instead of jQuery:

Remove this js:
```
function ScrollFollowerSprite (el1, follower1) {
    this.el = el1;
    this.follower = follower1 != null ? follower1 : null;
    this.isBlock = this.el.css('display') === 'block';
    //this.el.css('position', 'relative');
}
```

Add this css: 
```
.fc-cell-text {
  position: relative;
}
```

Before changes (notice the spikes and recalculates, takes about 200ms for 206 events)
<img width="1101" alt="screen shot 2017-06-27 at 10 40 59" src="https://user-images.githubusercontent.com/6674662/27578875-d20c0c1a-5b25-11e7-8629-65d5e9595e38.png">

After changes (notice the one recalculate, takes about 10ms for 206 events)
![screen shot 2017-06-27 at 10 38 02](https://user-images.githubusercontent.com/6674662/27578927-fb877f16-5b25-11e7-8b2e-202ac09452bc.png)

**Comment by MartijnWelker** [comment]
I also think destroying the calendar does an unnecessary amount of work, it tries to set widths and rebuild coord caches, but those steps aren't needed.

Areas marked in red are redundant 
![screen shot 2017-06-28 at 12 23 08](https://user-images.githubusercontent.com/6674662/27632817-2f409644-5bfd-11e7-8403-7e0d8c091683.jpg)

**Comment by arshaw** [comment]
queued for release

**Comment by arshaw** [comment]
Released in [v1.8.0](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.8.0)
